src/pages/landing/stocks/_dtrading.js

```diff
@@ -3,32 +3,34 @@ import styled from 'styled-components'
 import PropTypes from 'prop-types'
 import { graphql, useStaticQuery } from 'gatsby'
 import device from 'themes/device'
-import { Container, SectionContainer } from 'components/containers'
+import { Container, Flex, SectionContainer } from 'components/containers'
 import { Header, Text, QueryImage } from 'components/elements'
 
 const StyledSection = styled(SectionContainer)`
     background-color: var(--color-white);
+    padding: 40px 120px;
     @media ${device.tabletL} {
-        padding: 17px 0 4px 0;
+        padding: 40px 16px 32px;
     }
 `
+const Wrapper = styled(Container)`
+    width: 100%;
+`
 const Content = styled.div`
-    width: 60%;
+    width: 689px;
```

Comment:
Will this be an issue for mobile?

Reply:
no issue because the width will be changed on tabletL breakpoint.